src/pages/Order/Order.js

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from 'react';
+import './Order.scss';
+
+const Order = () => {
+  const [item, setItem] = useState([]);
+  const [user, setUser] = useState([]);
+
+  useEffect(() => {
+    fetch('./data/order.json')
+      .then(res => res.json())
+      .then(data => setItem(data.item));
+  }, []);
+
+  useEffect(() => {
+    fetch('./data/user.json')
+      .then(res => res.json())
+      .then(data => setUser(data.user));
+  }, []);
+
+  return (
+    <div className="order">
+      {user.map(el => (
```

Comment:
```suggestion
      {users.map(user => (
```
배열은 복수형태 입니다.
따라서 변수명도 ~~s의 형태가 좀 더 의미가 통할것 같고 map메서드의 콜백함수의 인자도 더 명확한 이름을 줄 수 있겠습니다.

Reply:
네 확인했습니다.